emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java

```diff
@@ -33,6 +33,10 @@ public class ImpactedClassMojo extends ImpactedMojo {
     @Parameter(property = "updateChecksums", defaultValue = "true")
     private boolean updateChecksums;
 
+    /** Parameter to determine whether fine RTS should be used. */
+    @Parameter(property = "enableFineRTS", defaultValue = "false")
+    private boolean enableFineRTS;
+
```

Comment:
Is this the right place for this parameter? That is, how come fineRTS is being enabled here, but not all the other options: method, hyrts etc? Also, is it currently possible for a user to select multiple CIA options? If so, how do you handle that case?